src/Common/ErrorCodes.cpp

```diff
@@ -642,7 +643,7 @@ namespace ErrorCodes
 
     std::string_view getName(ErrorCode error_code)
     {
-        if (error_code < 0 || error_code >= END)
+        if (error_code < 0 || error_code > END)
```

Comment:
Why?

Reply:
Because they've put the error code in the wrong place. It should be 724 and not require modifying anything else